docs/usage/model_config.md

```diff
@@ -658,7 +658,7 @@ except NameError as e:
 
 ## Hide Input in Errors
 
-_Pydantic_ shows the input value and type when it raises `ValidationError` during the validation.
+Pydantic shows the input value and type when it raises `ValidationError` during the validation.
```

Comment:
I think the italics here were a vestige of older pattern, just figured I'd address this while I was at it.